save-cloud-charts/save-cloud/README.md

```diff
@@ -17,10 +17,17 @@ api-gateway acts as an entrypoint and svc/gateway is actually a LoadBalancer.
 
   For example, for minikube and dev profile run `kubectl --context=minikube --namespace=save-cloud create secret generic db-secrets --from_literal=spring.datasource.username=<...> <...>`
 * **save-backend** and **save-demo** expects the following secrets to be set under the secret `s3-secrets` (`kubectl create secret generic s3-secrets <...>`)
-  * `s3-storage.endpoint`
-  * `s3-storage.bucketName`
-  * `s3-storage.credentials.accessKeyId`
-  * `s3-storage.credentials.secretAccessKey`
+  * For **backend** service
+    * `backend.s3-storage.bucketName`
```

Comment:
are you sure?
we have the following properties in `save-backend/src/main/resources/application.properties`:
```
backend.s3-storage.endpoint=${s3-storage.endpoint}
backend.s3-storage.bucketName=${s3-storage.bucketName}
backend.s3-storage.prefix=cnb/files
backend.s3-storage.credentials.accessKeyId=${s3-storage.credentials.accessKeyId}
backend.s3-storage.credentials.secretAccessKey=${s3-storage.credentials.secretAccessKey}
```

So `backend.s3-storage.*` are taken from `s3-storage.*`

Reply:
deleted unnecessary properties